docs/README.md

```diff
@@ -0,0 +1,97 @@
+Documentation Index
+===================
+
+This page serves as a categorized index for all `kops` documentation.
+
+
+Quick start
+-----------
+* [Getting started on AWS](aws.md)
+* [CLI reference](cli/kops.md)
+
+
+Overview
+--------
+
+* [CLI](#cli)
+* [Introspection](#introspection)
```

Comment:
Inspection perhaps? I mean: things you do when you look at the innards of a thing to debug it.